exampleSite/content/posts/theme-documentation-basics.md

```diff
@@ -221,6 +224,15 @@ Text to display in the footer section
   footer = "Text in footer"
 ```
 
+### Enable Front Matter
```

Comment:
Set heading for displaying content at homepage

Reply:
Not sure what you mean, so I reverted the documentation commit in c3d73a65272fc508c88ee814f6c9848f777ea08d